products/sle/main.pm

```diff
@@ -1129,6 +1129,8 @@ else {
     }
 }
 
+if (check_var('DESKTOP', 'minimalx')) { load_shutdown_tests; }
```

Comment:
The thought ist that all minimalx tests shut down in production.
If we set CREATE_HDD=1 for which I assume the load_create_hdd_tests will be called, we create  a hdd on production which we wanted to avoid.

Reply:
hmm could you provide a few verifications of different scenarios, how the actual schedule will look like?
I also have the feeling we miss something here and might end up with double scheduled modules.

try with `_EXIT_AFTER_SCHEDULE=1`

Reply:
We only have 3 scenarios for minimalx, none of them has the shutdown modules and I verified it on one.
Should be sufficient imho

Reply:
Because the next function is `load_common_opensuse_sle_tests` could you at least move it there and also test on openSUSE?